architecture/service_catalog/index.adoc

```diff
@@ -94,6 +94,13 @@ communicate with cluster service brokers and wait for them to perform their
 deprovisioning work. In normal circumstances, this may take several minutes or
 longer depending on the service.
 
+[NOTE]
+====
+If you remove a secret from your deployment environment, you must also remove
+the secret from the deployment environment tab. Otherwise, your rollout will
+fail.
```

Comment:
How about...

"If you delete a service binding used by a deployment, you must also remove any references to the binding secret from the deployment. Otherwise, the next rollout will fail."

Reply:
That is much better, thank you. :) I'll update the PR.